tests/unit/BlockStorage.d

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+
+    IO-performing tests for `agora.common.BlockStorage`
+
+    Create and store 1000 blocks and read again. And read 100 random.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module unit.BlockStorage;
+
+import agora.common.Amount;
+import agora.common.Block;
+import agora.common.BlockStorage;
+import agora.common.crypto.Key;
+import agora.common.Data;
+import agora.common.Hash;
+import agora.common.Transaction;
+import agora.consensus.Genesis;
+
+import std.algorithm.comparison;
+import std.file;
+import std.path;
+
+/// The maximum number of block in one file
+private immutable ulong MFILE_MAX_BLOCK = 100;
+
+///
+private void main ()
+{
+    const size_t count = 300;
+
+    string path = buildPath(getcwd, ".cache");
```

Comment:
Hmm.. this doesn't make the test parallelizable yet though, since multiple tests will end up overwriting each others data.

But we can fix this later.

Reply:
Yep, at this point I wanted to just move (and execute) the test.
We should go with something similar to Ocean (sandbox), but let's add a few tests first.